config.yaml

```diff
@@ -0,0 +1,30 @@
+options:
```

Comment:
I am trying to deploy and realized `server` is missing from this file.

Reply:
✅